### DISCUSSION TITLE
Segmentation Fault on Reconnect request from connlost

### DISCUSSION THREAD

**Discussion by sandippatelec** [eclipse-paho/paho.mqtt.c#1357] [Q&A] [answered]
MQTT Version - 1.3.12
Sample Code - src/samples/MQTTAsync_publish_time.c

The sample code getting crash while trying to reconnect the broker.

**Comment by icraggs** [comment, marked as answer]
I've fixed this is the develop branch - it will be in the next release. It was trying to print the cause when it was null.
